store/postgres/migrations/2019-05-15-215022_migrate_entities/up.sql

```diff
@@ -0,0 +1,173 @@
+create type deployment_schema_version as enum ('public', 'split');
+create type deployment_schema_state as enum ('ready', 'tables');
+
+alter table deployment_schemas
+  add column version   deployment_schema_version,
+  add column migrating bool not null default false,
+  add column state     deployment_schema_state not null default 'ready';
```

Comment:
I'm trying to understand this table and it's columns, but it's not the easiest thing to figure out betwen Rust code and SQL migrations. For example I can't figure out what `deployment_schema_state` does. If there was a place describing what this table does the code would be easier to review. Perhaps inside the diesel `table!` macro for it?

Reply:
I've added comments in `entities.rs` to try and clarify it. Let me know if it doesn't.